Library/Homebrew/utils/ruby.sh

```diff
@@ -1,3 +1,5 @@
+# shellcheck disable=SC2039
```

Comment:
I'd rather this is disabled on individual lines with an explanation of why it is.